src/urllib3/util/ssltransport.py

```diff
@@ -2,9 +2,26 @@
 import socket
 import io
 
+from urllib3.exceptions import ProxySchemeUnsupported
+
 SSL_BLOCKSIZE = 16384
 
 
+def validate_ssl_context_for_tls_in_tls(ssl_context):
+    """
+    Raises a RuntimeError if the provided ssl_context can't be used for TLS in
```

Comment:
Should be `ProxySchemeUnsupported` in the docs here, also should this be a private function? Maybe even add as a class method as `SSLTransport._validate_ssl_context()` so we don't have to do the optional import within `urllib3.util.ssl_`?